### PR TITLE
chore: open source stuff

### PR DESCRIPTION
**Pull request by WilsontheWolf**
Added codeowners
Added codeql

### CODE REVIEW

**Comment by WilsontheWolf** [comment]
Fine I'll load you up locally and run you

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      # ✏️ If the Autobuild fails above, remove it and uncomment the following
+      #    three lines and modify them (or add more) to build your code if your
+      #    project uses a compiled language
+
+      #- run: |
+      #     make bootstrap
+      #     make release
+
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v1
```
Comment (Chicken) [inline .github/workflows/codeql.yml:57]: Ew.
```suggestion
name: CodeQL

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:
  analyze:
    runs-on: ubuntu-latest
    permissions:
      security-events: write

    steps:
      - name: Checkout repository
        uses: actions/checkout@v2
.
      - name: Initialize CodeQL
        uses: github/codeql-action/init@v1

      - name: Perform CodeQL Analysis
        uses: github/codeql-action/analyze@v1
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,47 @@
...
+# DTO's
+src/dto/OffenceDto.js @WilsontheWolf
+src/dto/ResponseDto.js @KommentatorForAll
+src/dto/UserDto.js @WilsontheWolf @KommentatorForAll
+
+# Base Code
+src/loadModules.js @KommentatorForAll
+src/modUtil.js @WilsontheWolf
+src/notifcations.js @KommentatorForAll
+src/permissons.js @KommentatorForAll @WilsontheWolf
```
Comment (Chicken) [inline .github/CODEOWNERS:29]:
```suggestion
src/notificaitions.js @KommentatorForAll
src/permissions.js @KommentatorForAll @WilsontheWolf
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1,47 @@
...
+
+# Meta Stuff
+.github/workflows/ @Chicken
+.github/dependabot.yml @Chicken
+
+.dockerignore @Chicken
+.lintstagedrc.json @Chicken
+.prettierrc.json @Chicken
+docker-compose.yml @Chicken
+Dockerfile @Chicken
```
Comment (Chicken) [inline .github/CODEOWNERS:47]:
```suggestion
Dockerfile @Chicken
package.json @Chicken
```

**Review by Chicken — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1 +1,47 @@
...
+src/modUtil.js @WilsontheWolf
+src/notifcations.js @KommentatorForAll
+src/permissons.js @KommentatorForAll @WilsontheWolf
+src/roles.js @WilsontheWolf
+src/util.js @WilsontheWolf @KommentatorForAll
+src/xpHandling.js @KommentatorForAll
+src/index.js @WilsontheWolf @KommentatorForAll @Chicken
+
+# Config
+.env.example @Chicken
```
Comment (KommentatorForAll) [inline .github/CODEOWNERS:36]: Ain't this all of us too?
or at least me too, i mean, i committed almost half of the file

**Review by KommentatorForAll — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 4 above)
Comment (Chicken) [reply .github/CODEOWNERS:36]: Hmmh, true, imo just leave the .env file without owner, I still plan on revisiting the config system at somepoint.

Comment (WilsontheWolf) [reply .github/CODEOWNERS:36]: Yeah but chicken always yells at us about newlines

File: .github/CODEOWNERS (hunk 2 above)
Comment (WilsontheWolf) [reply .github/CODEOWNERS:29]: who needs "i"'s anyways?

**Review by KommentatorForAll — APPROVED** [review]
seems ok i guess

**Review by Chicken — APPROVED** [review]

**Pull request merged**